<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Waiting</class>
 <widget class="QWidget" name="Waiting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waiting for Players</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>

   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

   <item>
    <widget class="QLabel" name="waitingLabel">
     <property name="text">
      <string>Waiting for players...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="playersCountLabel">
     <property name="text">
      <string>Players: 0/4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="middleSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
